Sources/SwerverCore/Request/RequestMethod.swift

```diff
@@ -2,5 +2,5 @@ public enum RequestMethod: String {
     case get = "GET"
     case head = "HEAD"
     case options = "OPTIONS"
-    case other = "OTHER"
+    case invalid = "%$^ACHYT"
```

Comment:
Is this regex?

Reply:
this is just a random string to represent an invalid request method

Reply:
case invalid this has been removed